pkgs/tools/misc/figlet/default.nix

```diff
@@ -23,6 +23,12 @@ stdenv.mkDerivation rec {
     })
   ];
 
+  # TODO ATN: How to deal with `emcc' generating split binaries? Should `cp` and `install` be wrapped to also copy associated `.wasm` files? Should patches like this be added all over? Should `emcc` be wrapped to somehow combine the split binaries?
```

Comment:
Can you explain more about these split binaries?  When do you we need each part?  Wrapping `cp` and `install` doesn't sound feasible — there are many more ways to move files around.

Reply:
They are mentioning this feature: https://emscripten.org/docs/optimizing/Module-Splitting.html
emcc can generate multiple outputs : JS & WASM, it is not obvious that classical tooling would deal with the multiple outputs.